utility/src/main/java/org/oppia/util/parser/SvgSoftwareLayerSetter.kt

```diff
@@ -0,0 +1,37 @@
+package org.oppia.util.parser
+
+import android.graphics.drawable.PictureDrawable
+import android.widget.ImageView
+import com.bumptech.glide.load.DataSource
+import com.bumptech.glide.load.engine.GlideException
+import com.bumptech.glide.request.RequestListener
+import com.bumptech.glide.request.target.Target
+
+/**
+ * Listener which updates the [ImageView] to be software rendered, because [ ]/[Picture][android.graphics.Picture] can't render on a
+ * hardware backed [Canvas][android.graphics.Canvas].
+ */
+class SvgSoftwareLayerSetter : RequestListener<PictureDrawable> {
```

Comment:
Remove commented code in this file if not needed.

Reply:
Done.